app/views/users/show.html.erb

```diff
@@ -1,12 +1,36 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
-<!-- app/views/users/index.html.erb -->
-<h1>All Users</h1>
-<p>Here is a list of all users:</p>
-<!-- Implement code to display the list of users -->
-
-<!-- app/views/users/show.html.erb -->
-<h1>User Details</h1>
-<p>Here are the details for the user:</p>
-<!-- Implement code to display the details of a single user -->
-
+<h1 class='title'>Welcome to Blog-Haven</h1>
+<div class= "container">
+<% if @user %>
+    <div class='user'>
+      <div class='avatar'>
+        <%= image_tag(@user.photo) %>
+      </div>
+    <div><h2><%= @user.name %></h2></div>
+    <div><p>No of Posts:</p></div>
```

Comment:
- [ ] KIndly make sure to display post counts on user profiles to enhance user experience and engagement. Let's make this improvement to achieve our project milestone successfully.

   - ## PROVIDED DESIGN 
  ![image](https://github.com/eudondian/Blog-Haven/assets/87197412/93d32fce-feb7-4257-a8bb-7b1da92e14ec)

   - ### YOUR DESIGN
![image](https://github.com/eudondian/Blog-Haven/assets/87197412/61c0a57e-db0b-4db0-84de-848b0a6513ff)